test
====

Major applications of RNA-seq data include studies of how the transcriptome is modulated at the levels of gene expression and RNA processing, and how these events are related to cellular identity, environmental condition, and/or disease status. While many excellent tools have been developed to analyze RNA-seq data, these generally have limited efficacy for annotating 3' UTRs. As illustrated below, existing assembly strategies often fragment long 3' UTRs, and importantly, none of the algorithms in popular use can apportion data into tandem 3' UTR isoforms, which are frequently generated by alternative cleavage and polyadenylation (APA). Consequently, it is often not possible to identify patterns of differential APA using existing assembly tools. 

![motivation image](./motivation.png)

To address these limitations, we developed Isoform Structural Change Model (IsoSCM),  a new method for transcript assembly that incorporates change-point analysis to improve the 3' UTR annotation process.

![changepoint image](./changepoints.png)

![differential image](./differential_usage.png)

usage<sub>brain</sub> = 1 - ( cov<sub>downstream,brain</sub> / cov<sub>upstream,brain</sub> ) &cong; 0.68

usage<sub>liver</sub> = 1 - ( cov<sub>downstream,liver</sub> / cov<sub>upstream,liver</sub> ) &cong; 0.91

&Delta;usage<sub>brain-liver</sub> = usage<sub>brain</sub>-usage<sub>liver</sub> &cong; -0.23
